misc/python/materialize/cli/ci_coverage_pr_report.py

```diff
@@ -0,0 +1,90 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+import re
+import subprocess
+import sys
+from collections import OrderedDict
+from itertools import groupby
+from typing import Dict, List
+
+Coverage = Dict[str, OrderedDict[int, int]]
+SOURCE_RE = re.compile(
+    "^/var/lib/buildkite-agent/builds/buildkite-.*/materialize/tests/(.*$)"
+)
+
+
+def find_interesting_lines() -> Coverage:
```

Comment:
I think this is misnamed -- what is "interesting" in this case? I guess "get_added_lines" will be more to the point?